source/content/guides/new-relic/03-monitor-new-relic.md

```diff
@@ -45,11 +45,19 @@ For more information on using New Relic&reg;'s features, we encourage you to rev
 
 - [Troubleshooting WordPress Performance with New Relic&reg; Performance Monitoring](https://pantheon.io/blog/troubleshooting-wordpress-performance-new-relic)
 
-## Configure Ping Monitors for Availability
+## Configure Ping Monitors (Synthetics) for Availability
 
-New Relic provides a free availability monitoring service within their Synthetics tool suite at the Lite service level. This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.
+New Relic provides an availability monitoring service within their Synthetics tool suite which can be enabled by [contacting support](/guides/support/contact-support/). This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.
 
-Pantheon can provide New Relic Ping monitoring for free as part of the service. However, more advanced monitoring, including full browser testing and scripted interactions, is only available to customers on an annual contract and requires an additional cost. Contact our [sales team](https://pantheon.io/contact-us) or your dedicated account manager for more information.
+Pantheon can provide New Relic Synthetics ping monitoring as part of the service with a usage quota limited to 10K Synthetics checks per month for each Pantheon workspace.
```

Comment:
```suggestion
Pantheon caps usage of ping monitoring at 10K Synthetics checks per month for each Pantheon workspace.
```

Reply:
@stevector We need to clarify this: the quota doesn't include ping monitors (customers can use unlimited ping monitors); the limit only applies to non ping monitors. I'd propose something like this:
```suggestion
Pantheon provides unlimited New Relic Synthetics ping monitoring as part of the service. Still, there's a quota for all other monitor types (simple browser, scripted browser, and scripted API monitors). This quota is set at 10K Synthetics monthly checks for each Pantheon account, so it's crucial to manage it effectively.
```